### ISSUE TITLE
JPA data source connection leak

### ISSUE THREAD

**Issue by javolek** [paulcwarren/spring-content#272]
There seems to be data source connection leak in the class `internal.org.springframework.content.jpa.io.DelegatingBlobResourceLoader`. On the Line 42 of the class file the connection is created that is never closed:
```
java.lang.Exception: Apparent connection leak detected
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:128)
	at internal.org.springframework.content.jpa.io.DelegatingBlobResourceLoader.getResource(DelegatingBlobResourceLoader.java:42)
	at internal.org.springframework.content.jpa.repository.DefaultJpaStoreImpl.getResource(DefaultJpaStoreImpl.java:46)
	at internal.org.springframework.content.jpa.repository.DefaultJpaStoreImpl.setContent(DefaultJpaStoreImpl.java:106)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
```

**Comment by paulcwarren** [comment]
Looks like it, yes.  We will address this in the next release.  Thanks for the report.

**Comment by paulcwarren** [comment]
Fixed with [this commit](https://github.com/paulcwarren/spring-content/commit/929c0ad5ad51f435ef2c3e193335b084f8860e94).  Will be released in 1.1.0.M3.
